### DISCUSSION TITLE
[Feature] Have the abillity to use xmp notes and notes photos to 0 star to 5 star to sort photo

### DISCUSSION THREAD

**Discussion by yodatak** [immich-app/immich#9355] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want to be able to make a quick sort with star to know with photo i want to work on dartable from the raw imich would permit a got way to sort photo simple do you think its possible ?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3619
